frontend/storage/configureStore.ts

```diff
@@ -25,7 +26,12 @@ function createMockGamesVisibility(): Map<number, boolean> {
 function createMockGames(): Map<number, Game> {
 	let mockData: Map<number, Game> = new Map();
 	for (let i = 0; i < 20; i++) {
-		const mockGame = new Game({ id: i, name: `Super Metroid: ${i}` });
+		const mockGame = new Game({
+			id: i,
+			name: `Super Metroid: ${i}`,
+			series: 'Metroid',
+			img: SuperMetroidCover,
+		});
 		mockData.set(i, mockGame);
```

Comment:
You don't really gain anything by first assigning the game to a constant and then passing the constant to `set()` - the name of the function (`createMockGames()`) makes it obvious what's happening. I suggest to just collapse the two into one statement.

As a side note, next time you add any more mock generation create a new function that creates a full mock store in a separate file :).